extensions/guacamole-auth-cas/src/licenses/LICENSE

```diff
@@ -0,0 +1,294 @@
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+==============================================================================
+
+APACHE GUACAMOLE SUBCOMPONENTS
+
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
+terms and conditions of the following licenses.
+
+
+AOP Alliance (http://aopalliance.sourceforge.net/)
+--------------------------------------------------
+
+    Version: 1.0
+    From: 'AOP Alliance' (http://aopalliance.sourceforge.net/members.html)
+    License(s):
+        Public Domain (bundled/aopalliance-1.0/LICENSE)
+
+Commons Codec (http://commons.apache.org/codec/)
+------------------------------------------------
+
+    Version: 1.4
+    From: 'Apache Software Foundation' (http://www.apache.org)
+    License(s):
+        Apache v2.0 (bundled/commons-codec/LICENSE-2.0.txt)
+
+
+Google Guice (https://github.com/google/guice)
+----------------------------------------------
+
+    Version: 3.0
+    From: 'Google Inc.' (http://www.google.com/)
+    License(s):
+        Apache v2.0 (bundled/guice-3.0/COPYING)
+
+Javax Servlet API (http://servlet-spec.java.net/)
+-------------------------------------------------
+
+    Version: 3.0.1
+    From: 'GlassFish Community' (http://servlet-spec.java.net/)
+    License(s):
+        CDDL 2 + GPLv2 with ClassPath Exception (bundled/javax.servlet-api/LICENSE)
+
+
+JSR-330 / Dependency Injection for Java (http://code.google.com/p/atinject/)
+----------------------------------------------------------------------------
+
+    Version: 1
+    From: 'JSR-330 Expert Group' (https://jcp.org/en/jsr/detail?id=330)
+    License(s):
+        Apache v2.0 (bundled/javax.inject-1/LICENSE-2.0.txt)
+
+JUnit (http://junit.org)
+------------------------
+
+    Version: 4.8.2
+    From: 'JUnit' (http://junit.org)
+    License(s):
+        CPL v1.0 (bundled/junit/LICENSE-1.0.txt)
+
+LOG4J (http://logging.apache.org/log4j/1.2/)
+--------------------------------------------
+
+    Version: 1.2.15
+    From: 'Apache Software Foundation' (https://www.apache.org)
+    License(s):
+        Apache v2.0 (bundled/log4j/LICENSE-2.0.txt)
+
+slf4j (http://www.slf4j.org)
+----------------------------
+
+    Version: 1.7.1
+    From: 'Quality Open Software' (http://www.qos.ch)
+    License(s):
+        MIT (bundled/slf4j/LICENSE)
+
+Spring Framework (https://projects.spring.io/spring-framework/)
+---------------------------------------------------------------
+
+    Version: 3.0.5.RELEASE
+    From: 'Spring by Pivotal' (https://spring.io)
+    License(s):
+        Apache v2.0 (bundled/springframework/LICENSE-2.0.txt)
+
+XML Security (http://santuario.apache.org)
```

Comment:
What pulls this in? I don't see this in the dependency list produced by Maven either.